css/src/components/hero.scss

```diff
@@ -143,13 +189,25 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 	&.hero-image:dir(rtl) {
 		direction: ltr;
 
-		.hero-content {
+		.hero-content,
+		.hero-details {
 			direction: rtl;
 
 			@include desktop {
 				padding-inline-end: unset;
 				padding-inline-start: $hero-padding-sm;
 			}
 		}
+
+		&.hero-with-details {
+			direction: rtl;
+		}
+
+		&.gradient-border-right::before {
```

Comment:
This comment pertains to this whole section:

As far as I can tell, even prior to this PR we aren't adequately targeting right to left scenarios with our heroes, which we don't want to flip in the case that the text direction is rtl, but I think the following should fix that:

```css

	// Set direction to prevent image flip for rtl
	[dir='rtl'] &.hero-image,
	&.hero-image:dir(rtl) {
		direction: ltr;

		.hero-content,
		.hero-details {
			direction: rtl;

			@include desktop {
				padding-inline-end: unset;
				padding-inline-start: $hero-padding-sm;
			}
		}

		&.hero-with-details {
			direction: rtl;
		}

		&[class^='gradient-border-'] {
			direction: ltr;
		}
	}
```

Reply:
Ill get that in.

Reply:
@BenDMyers  Context.

Reply:
Ah, fair enough- resolving my other comment.